classes/Dispatcher.php

```diff
@@ -270,30 +270,70 @@ private function getRequest()
     {
         return $this->request;
     }
+    
+    /**
+     * Sets and returns the default controller.
+     *
+     * @param int $frontControllerType The front controller type
+     * @param Employee|null $employee The current employee
+     *
+     * @return string
+     */
+    private function getDefaultController($frontControllerType, Employee $employee = null)
+    {
+        switch ($frontControllerType) {
+            case self::FC_ADMIN:
+                // If there is an employee with a default tab set
+                if (null !== $employee) {
+                    $tabClassName = $employee->getDefaultTabClassName();
+                    if (null !== $tabClassName) {
+                        $defaultController = $tabClassName;
+                    }
+                }
+                // Default
+                $defaultController = 'AdminDashboard';
```

Comment:
Be careful, now you removed the `return` statement, this default value will override `$defaultController = $tabClassName;`

To avoid this, you can move this line after the `case`:
```php
case self::FC_ADMIN:
    // Default
    $defaultController = 'AdminDashboard';
    [...]
```